src/components/css/Save.css

```diff
@@ -0,0 +1,27 @@
+.Save-Patch {
+  font-family: "Toxigenesis";
+  position: absolute;
+  background-color: var(--background-body-color);
+  z-index: 9999;
+  border: 5px solid var(--general-border-color);
+  border-radius: 15px;
+  width: 85%;
+  height: 70%;
+  margin-bottom: 190%;
```

Comment:
If you are using position-absolute, then use `top` to position the modal, instead of a random margin-bottom